.clang-format

```
@@ -0,0 +1,117 @@
+Language: Cpp
+BasedOnStyle: LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignEscapedNewlines: Left
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: true
+AllowShortCaseLabelsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: All
+AllowShortIfStatementsOnASingleLine: true
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: AllDefinitions
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: true
+BinPackParameters: true
```

```suggestion
BinPackParameters: false
```
I'd disable bin packing so we have:
```
RobotMemory::RobotMemory(fawkes::Configuration *     config,
                         fawkes::Logger *            logger,
                         fawkes::Clock *             clock,
                         fawkes::MongoDBConnCreator *mongo_connection_manager,
                         fawkes::BlackBoard *        blackboard)

```
instead of
```
RobotMemory::RobotMemory(fawkes::Configuration *config, fawkes::Logger *logger,
                         fawkes::Clock *clock, fawkes::MongoDBConnCreator *mongo_connection_manager,
                         fawkes::BlackBoard *blackboard)
```